test/validate_v1_v2_equality.ts

```diff
@@ -1,36 +1,244 @@
-import { getCardsJson, getCardsV2Json } from "../src/index";
+import { getCardsJson, getCardSetsV2Json, getCardsV2Json, getPacksJson, getPrintingsV2Json, textToId } from "../src/index";
 import chai = require('chai');
 const expect = chai.expect;
 
-describe('card text matches between versions', () => {
+describe('Cards v1/v2 equality', () => {
   const v1Cards = getCardsJson();
-  const v1Text = new Map<string, string>();
-  const v1StrippedText = new Map<string, string>();
+  const v1CardsByTitle = new Map<string, any>();
   v1Cards.forEach(c => {
-    v1Text.set(c.title, c.text);
-    v1StrippedText.set(c.title, c.stripped_text);
+    v1CardsByTitle.set(c.title, c);
   });
+  const v2CardsByTitle = new Map<string, any>();
```

Comment:
Any particular reason for separate maps instead of a single map of id to card object?